packages/consumption/src/consumption/CoreErrors.ts

```diff
@@ -23,6 +23,25 @@ class Attributes {
         return new CoreError("error.consumption.attributes.successionPeerIsNotOwner", "The peer of the succeeded attribute is not its owner. This may be an attempt of spoofing.");
     }
 
+    public predecessorSourceAttributeInvalid() {
```

Comment:
How about `predecessorSourceAttributeIsNotRepositoryAttribute`? The name is unnecessarily vague. Longer names are alright, if that's what's bothering you :-)

Reply:
Same for `successorSourceAttributeInvalid`.